webbpsf/utils.py

```diff
@@ -429,3 +429,40 @@ def measure_strehl(HDUlist_or_filename=None, ext=0, slice=0, center=None, displa
         print("  Strehl ratio: {0:.3f}".format(strehl))
 
     return strehl
+
+
+### Helper routines for display customization: ###
+
+def annotate_ote_entrance_coords(self, ax):
+    """ Draw OTE V frame axes on first optical plane """
+    ax.set_ylabel('OTE V3 [m]')
+    ax.set_xlabel('OTE V2 [m]')
+    color='yellow'
+    loc = 3
+    ax.arrow(-loc,-loc, .2, 0, color=color, width=0.005)
+    ax.arrow(-loc,-loc, 0, .2, color=color, width=0.005)
+    ax.text(-loc, -loc+0.4, '+V3', color=color,size='small', horizontalalignment='center',verticalalignment='bottom')
+    ax.text(-loc+0.4, -loc, '+V2', color=color,size='small', horizontalalignment='left',verticalalignment='center')
+
+def annotate_sky_pupil_coords(self, ax):
+    """ Draw OTE V frame axes projected onto the sky"""
+    #ax.set_ylabel('OTE V3 [m]')
+    #ax.set_ylabel('OTE V2 [m]')
+    color='yellow'
+    loc = 2.9
+    ax.arrow(-loc+0.5,-loc, -.2, 0, color=color, width=0.005)
+    ax.arrow(-loc+0.5,-loc, 0, .2, color=color, width=0.005)
+    ax.text(-loc+0.5, -loc+0.3, '+V3 on sky', color=color,size='small', horizontalalignment='center',verticalalignment='bottom')
+    ax.text(-loc+0.5+0.3, -loc, '+V2 on sky', color=color,size='small', horizontalalignment='left',verticalalignment='center')
+
+
+    if 0:
```

Comment:
If this is a no-op, maybe axe it? always available in the git history if it turns out to be needed